base_gs1_barcode/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2012-2014 Numérigraphe SARL.
+{
+    'name': 'GS1 Barcode API',
+    'summary': 'Decoding API for GS1-128 (aka UCC/EAN-128) and GS1-Datamatrix',
+    'version': '11.0.1.0.0',
+    'author': 'Numérigraphe, Odoo Community Association (OCA)',
+    'website': 'http://numerigraphe.com',
+    'category': 'Generic Modules/Inventory Control',
+    'depends': [
+        'sale',
```

Comment:
Why depending on `sale`?

Reply:
Because parent menu prod_config_main is in sale module.
¿Whe can I put it?

Reply:
Yeah, but that menu seems a bit... random, and security is not in line with the expected behavior. IMO, you can grant read access to employees, but only edit, creation and removal to system users, and place the menu in system section then.